src/containers/Project/ProjectAssignmentDetails.tsx

```diff
@@ -43,14 +43,28 @@ const ProjectAssignmentDetails = ({
   const queryClient = useQueryClient();
   const [open, setOpen] = useState(false);
   const [enteredTag, setEnteredTag] = useState<string | null>('');
+  const projectDesignationByAssignmentProps = {
+    projectAssignmentId,
+    page: 1,
+    limit: 10,
+  };
+  const {
+    projectDesignationData,
+    projectDesignationLoading,
+    addProjectDesignationByAssignmentAction,
+    deleteProjectDesignationByAssignmentAction,
+  } = useProjectDesignationByAssignment(projectDesignationByAssignmentProps);
 
-  const { projectDesignationData, projectDesignationLoading } =
-    useProjectDesignationByAssignment(projectAssignmentId, 1, 10);
   const { tagData, istagLoading } = useTag();
-  const { addProjectDesignationByAssignmentAction } =
-    useAddProjectDesignationByAssignment();
-  const { deleteProjectDesignationByAssignmentAction } =
-    useDeleteProjectDesignationByAssignment();
+
+  const [alertOpen, setAlertOpen] = useState(false);
+  const [alertSeverity, setAlertSeverity] = useState<
+    'success' | 'error' | 'info' | 'warning'
```

Comment:
create this type outside of component
`type AlertType = 'success' | 'error' | 'info' | 'warning'`
and use it like this
`useState<AlertType>`